### PR TITLE
Added rule for multiple variables

### PR DESCRIPTION
**Pull request by wxu-aa**
We should add a rule to enforce adding sprintf positional markers to the variables if a string contains more than a single marker:

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+function getReportNode(argNode, node) {
+    return argNode || node.callee;
+}
+
+function isStringLiteral(node) {
+    return node && node.type === 'Literal' && typeof node.value === 'string';
+}
+
+const hasMultipleVaribles = str => {
+    if (str.match(/%[sd]/gm) && str.match(/%\d+\$(s|d)/gm)) {
```
Comment (SBoudrias) [inline lib/utils.js]: Why are we doing the first check here?

We can also simplify the second regex to `/%\d+\$[sd]/gm`

**Comment by SBoudrias** [comment]
LGTM thanks :D

File: lib/utils.js (hunk 1 above)
Comment (wxu-aa) [reply lib/utils.js]: Fixed👍

**Comment by wxu-aa** [comment]
@SBoudrias 

Is it fine to merge or release to npm ? Thanks for your double checking👍

**Pull request merged**